### DISCUSSION TITLE
Oqtane

### DISCUSSION THREAD

**Discussion by markdav-is** [markdav-is/Hexstor#9] [General]
To focus on making the game, and not inventing a web framework for user management, settings, data-access, state-control etc we are using Oqtane framework.  We'll build modules and compose pages by adding those modules to pages.  To get started, be sure to initialize the oqtane submodule to the proper release specified in gitmodules (v6.0.1 at the time of this post).

most importantly: you must build the oqtane.sln in the submodule on your local machine to have those libraries available.

**Comment by DrCord** [comment]
including a command to help someone build the `oqtane.sln in the submodule on your local machine` somewhere in the repo would be very helpful!
